Yoast/wordpress-seo
# Merge Gutenberg sidebar feature branch into trunk

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices

* Add a feature flag for the Gutenberg sidebar. Use `define('YOAST_FEATURE_GUTENBERG_SIDEBAR', true );` in your `wp-config.php` to see the sidebar. Without that flag, the sidebar will not be registered.
* Introduces Redux actions and reducers that will be used for the sidebar. 

## Test instructions

This PR can be tested by following these steps:

* Use the feature flag to toggle the visibility of the Gutenberg sidebar.
* Make sure nothing else breaks.

## Quality assurance
